cli/pcluster/configure/networking.py

```diff
@@ -109,16 +119,17 @@ def _create(self, vpc_id, vpc_cidr, subnet_cidrs, internet_gateway_id, compute_s
 class PublicPrivateNetworkConfig(BaseNetworkConfig):
     """The publicprivate configuration that creates one public subnet for master and one private subnet for compute."""
 
-    def __init__(self):
+    def __init__(self, availability_zones=None):
         super(PublicPrivateNetworkConfig, self).__init__(
             config_type="Master in a public subnet and compute fleet in a private subnet",
             template_name="public-private",
             stack_name_prefix="pubpriv",
+            availability_zones=availability_zones,
         )
 
     def get_cfn_parameters(self, vpc_id, internet_gateway_id, public_cidr, private_cidr):
         """Create cloudformation-compatible stack parameter given the variables."""
-        parameters = super(PublicPrivateNetworkConfig, self)._get_cfn_parameters(vpc_id, internet_gateway_id)
+        parameters = self._get_cfn_parameters(vpc_id, internet_gateway_id)
```

Comment:
I would prefer to leave the `super`, even if this works it looks like a recursion :p